UPDATING.md

```diff
@@ -23,6 +23,16 @@ This file documents any backwards-incompatible changes in Superset and
 assists people when migrating to a new version.
 
 ## Next
+- [12072](https://github.com/apache/incubator-superset/pull/12072): Security simplification (SIP-19) permission view name renamed to `Query` with `can_read`, `can_write`. Automatic db migration to new set of permissions for all existing roles.
+- [12036](https://github.com/apache/incubator-superset/pull/12036): Security simplification (SIP-19) permission view name renamed to `Database` with `can_read`, `can_write`. Automatic db migration to new set of permissions for all existing roles.
+- [12012](https://github.com/apache/incubator-superset/pull/12036): Security simplification (SIP-19) permission view name renamed to `Dashboard` with `can_read`, `can_write`. Automatic db migration to new set of permissions for all existing roles.
+- [12061](https://github.com/apache/incubator-superset/pull/12061): Security simplification (SIP-19) permission view name renamed to `Log` with `can_read`, `can_write`. Automatic db migration to new set of permissions for all existing roles.
+- [12000](https://github.com/apache/incubator-superset/pull/12000): Security simplification (SIP-19) permission view name renamed to `Dataset` with `can_read`, `can_write`. Automatic db migration to new set of permissions for all existing roles.
+- [12014](https://github.com/apache/incubator-superset/pull/12014): Security simplification (SIP-19) permission view name renamed to `Annotation` with `can_read`, `can_write`. Automatic db migration to new set of permissions for all existing roles.
+- [11981](https://github.com/apache/incubator-superset/pull/11981): Security simplification (SIP-19) permission view name renamed to `Chart` with `can_read`, `can_write`. Automatic db migration to new set of permissions for all existing roles.
+- [11853](https://github.com/apache/incubator-superset/pull/11853): Security simplification (SIP-19) permission view name renamed to `ReportSchedule` with `can_read`, `can_write`. Automatic db migration to new set of permissions for all existing roles.
+- [11856](https://github.com/apache/incubator-superset/pull/11856): Security simplification (SIP-19) permission view name renamed to `CssTemplate` with `can_read`, `can_write`. Automatic db migration to new set of permissions for all existing roles.
+- [11764](https://github.com/apache/incubator-superset/pull/11764): Security simplification (SIP-19) permission view name renamed to `SavedQuery` with `can_read`, `can_write`. Automatic db migration to new set of permissions for all existing roles.
```

Comment:
To make these stand out as a single entry in the list, would it make sense to group these under one bullet Stating that SIP-19 security simplification has been done with the sub bullets referencing the individual perms?

Reply:
makes sense, updated @villebro what do you think?